modules/serverless-kafka-broker-with-isolated-dataplane.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * serverless/eventing/brokers/serverless-using-brokers.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-broker-with-isolated-dataplane_{context}"]
+= Creating a Knative broker for Apache Kafka that uses an isolated data plane
+
+:FeatureName: The Knative Broker implementation for Apache Kafka with isolated data plane
+include::snippets/technology-preview.adoc[leveloffset=+2]
+
+The Knative Broker implementation for Apache Kafka has 2 planes:
+
+Control plane:: Consists of controllers that talk to Kubernetes API, watch for custom objects and manage the data plane.
+
+Data plane:: The collection of components that listen for incoming events, talk to Apache Kafka and  send events to the event sinks. The Knative Broker implementation for Apache Kafka data plane is where the events flow, and consists of `kafka-broker-receiver` and `kafka-broker-dispatcher` deployments.
+
+When using a Broker class of `Kafka`, the Knative Broker implementation for Apache Kafka uses a shared data plane. This means that the `kafka-broker-receiver` and `kafka-broker-dispatcher` deployments in the `knative-eventing` namespace are used for all Apache Kafka Brokers in the cluster.
+
+However, when using a Broker class of `KafkaNamespaced`, the Apache Kafka broker controller creates a new data plane for each namespace where a broker exists. This data plane is used by all `KafkaNamespaced` brokers in that namespace.
+
+This provides isolation between the data planes, so that the `kafka-broker-receiver` and `kafka-broker-dispatcher` deployments in the user namespace are only used for the broker in that namespace.
+
+[IMPORTANT]
+====
+As a consequence of having separate data planes, this security feature creates more deployments and uses more resources. Unless you have such isolation requirements, it is preferable to use a *regular* Broker with a class of `Kafka`.
```

Comment:
```suggestion
As a consequence of having separate data planes, this security feature creates more deployments and uses more resources. Unless you have such isolation requirements, do not use a *regular* Broker with a class of `Kafka`.
```
It's always better to be opinionated and tell the user what to do or not do rather than waffling and saying `preferable`, imo.